aws.dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM maven:3-jdk-8-alpine as java-build
+
+COPY kinesis-client /opt/sites/rig/kinesis-client
```

Comment:
here you also copy the jinterface jar that is used during compilation and also later at runtime:
```
KINESIS_OTP_JAR=/opt/sites/rig/kinesis-client/local-maven-repo/org/erlang/otp/jinterface/1.8.1/jinterface-1.8.1.jar
```

However, I'd prefer having the runtime compiled with Java support and using the runtime jinterface jar for both compilation and runtime. In that case, the KINESIS_OTP_JAR variable doesn't have to be set, because RIG is expecting the jinterface jar at the right place (as defined by the spec).

Reply:
We've settled on not compiling it for now, but instead using the checked-in version of JInterface (OtpErlang.jar). Because of that, I feel that we should provide the right default for the path inside the image; right now, when setting `KINESIS_ENABLED=1`, you see the following error:

```plain
"Could not find OtpErlang.jar for JInterface 1.8.1 at /opt/sites/rig/lib/jinterface-1.8.1/priv/OtpErlang.jar. Does your Erlang distribution come with Java support enabled?"
```

I propose that we simply use `ENV` in the dockerfile to point to the right path, so the user doesn't have to set this later on:

```Dockerfile
ENV KINESIS_OTP_JAR=/opt/sites/rig/kinesis-client/local-maven-repo/org/erlang/otp/jinterface/1.8.1/jinterface-1.8.1.jar
```